DashAI/front/src/components/experiments/SplitDatasetRows.jsx

```diff
@@ -56,79 +58,137 @@ function SplitDatasetRows({
       color: "#bbb",
     },
   };
-  const [rowsPartitionsError, setRowsPartitionsError] = useState(false);
-  const [rowsPartitionsErrorText, setRowsPartitionsErrorText] = useState("");
+  const [randomSplitError, setRandomSplitError] = useState(false);
+  const [randomSplitErrorText, setRandomSplitErrorText] = useState("");
+  const [manualSplitError, setManualSplitError] = useState(false);
+  const [manualSplitErrorText, setManualSplitErrorText] = useState("");
 
   const handleSplitTypeChange = (event) => {
     const newType = event.target.value;
     setSplitType(newType);
 
     if (newType === SPLIT_TYPES.PREDEFINED) {
-      setRowsPartitionsError(false);
       setSplitsReady(true);
     }
     if (newType === SPLIT_TYPES.RANDOM) {
-      setSplitType(newType);
-      setRowsPartitionsPercentage({ train: 0.6, test: 0.2, validation: 0.2 });
+      const newSplit = { train: 0.6, test: 0.2, validation: 0.2 };
+      setRowsPartitionsPercentage(newSplit);
+
+      // Validate the random split
+      const hasZero =
+        newSplit.train === 0 ||
+        newSplit.validation === 0 ||
+        newSplit.test === 0;
+      const sumsToOne = checkSplit(
+        newSplit.train,
+        newSplit.validation,
+        newSplit.test,
+      );
+
+      if (hasZero) {
+        setRandomSplitErrorText("All splits must be greater than 0");
+        setRandomSplitError(true);
```

Comment:
I would let Validation or Test be 0, you can train with 100% data if you want, but train split can't be 0, something like:


```suggestion
      const hasZeroTrain = newSplit.train === 0;
      const sumsToOne = checkSplit(newSplit.train, newSplit.validation, newSplit.test);
      
      if (hasZeroTrain) {
        setRandomSplitErrorText("Train split must be greater than 0");
        setRandomSplitError(true);
```

I think huggingface backend allows this

Reply:
if that's the case then should it be allowed to train with less than 100% of the dataset?. E.g.

- train = 0.6
- test = 0,2
- val = 0